src/scripts.js

```diff
@@ -344,9 +348,38 @@ function populatePantry() {
       <tr>
         <td>${ingredientData.name}</td>
         <td>${item.amount}</td>
+        <td class="button-column">
+          <button class="round-buttons">+</button>
+          <button class="round-buttons">-</button>
+        </td>
       </tr>`
   })
-  console.log(currentUser.name)
+}
+
+function populateDropdown() {
+  const ingData = ingredientsData.filter(ingredient => {
+    return typeof ingredient.name === 'string'
+  })
+  const sortedIngredients = ingData.sort((a, b) => {
+    if(((typeof a.name) === 'string') && ((typeof b.name) === 'string')) {
+      let nA = a.name.toLowerCase();
+      let nB = b.name.toLowerCase();
+       if(nA < nB) {
+        return -1
+      } else if(nA > nB) {
+        return 1
+      }else {
+        return 0
+      }
+    }
+  });
+
+  console.log(sortedIngredients)
+  const dropdownData = sortedIngredients.forEach(ingredient => {
+    const optionElement = `<option value="${ingredient.id}">${ingredient.name}</option>`;
+    dropdownElement.innerHTML += optionElement; 
+  })
+  return dropdownData
 }
```

Comment:
super smart to go with a dropdown menu for the ingredients as opposed to input fields. Now we just gotta get this functional!